packages/common/src/schemas/index.ts

```diff
@@ -1,149 +1,2 @@
-// TODO(nkang) - convert to TS
```

Comment:
idk why but this was previously a "barrel file" but there was actually just stuff in here